dev-qt/qtsql/qtsql-4.8.7-r1.ebuild

```diff
@@ -37,7 +38,7 @@ QT4_TARGET_DIRECTORIES="
 multilib_src_configure() {
 	local myconf=(
 		$(qt_native_use freetds  sql-tds    plugin)
-		$(qt_use        mysql    sql-mysql  plugin) $(use mysql && echo "-I${EPREFIX}/usr/include/mysql -L${EPREFIX}/usr/$(get_libdir)/mysql")
+		$(qt_use        mysql    sql-mysql  plugin) $(use mysql && echo -mysql_config "/usr/bin/${CHOST}-mysql_config")
```

Comment:
no `EPREFIX`?

Reply:
Fixed in 2a44024ff26b8d8c8ad4c160c55411b9128ec30b